integration-tests/src/test/java/oracle/kubernetes/operator/ItUsabilityOperatorHelmChart.java

```diff
@@ -130,9 +133,15 @@ public void testOperatorCreateDeleteCreate() throws Exception {
   }
 
   /**
-   * Negative test: Helm will install 2 operators, with same namespace, second operator should fail.
+   * Negative test: 
+   * Install Operator usab-1 with name namespace op-usab-ns
+   * Install Operator usab-2 with same namesapce op-usab-ns
+   * Second operator MUST fail to intstall with following Exception  
+   * Error: rendered manifests contain a resource that already exists. 
+   * Unable to continue with install: existing resource conflict: kind: Secret, 
+   * namespace: usab-opns-1, name: weblogic-operator-secrets
    *
-   * @throws Exception exception
+   * @throws Exception when Second Operator installtion does not fail
```

Comment:
"second" should not have a capital
"operator" should not have a capital
"installation" spelled wrong

Reply:
done